gulpfile.js

```diff
@@ -122,54 +101,9 @@ gulp.task('copyTypings', function() {
     .pipe(gulp.dest(paths.build))
 });
 
-// Lints the source and test files
-gulp.task('lint', function() {
-  let filesToLint = _.clone(paths.src.concat(paths.tests));
-
-  // Don't lint the hand-crafted TypeScript typings file
-  filesToLint.push('!src/index.d.ts');
-
-  return gulp.src(filesToLint)
-    .pipe(tslint())
-    .pipe(tslint.report({
-      summarizeFailureOutput: true
-    }));
-});
-
-// Runs the test suite
-gulp.task('test', function() {
-  merge(
-    // Copy compiled source and test files
-    gulp.src(paths.tests.concat(paths.src), {base: '.'})
-      .pipe(testProject())
-      .pipe(gulp.dest(paths.testBuild)),
-    // Copy compiled database files
-    gulp.src(paths.build + 'database/**/*')
-      .pipe(gulp.dest(paths.testBuild + 'src/database/')),
-    // Copy test resources
-    gulp.src(paths.resources, {base: '.'})
-      .pipe(gulp.dest(paths.testBuild))
-  ).on('finish', function() {
-    return gulp.src([paths.testBuild + 'src/**/*.js', '!' + paths.testBuild + 'src/database/**/*'])
-      .pipe(istanbul())
-      .pipe(istanbul.hookRequire())
-      .on('finish', function() {
-        return gulp.src(paths.testRunner)
-          .pipe(mocha({
-            reporter: 'spec',
-            timeout: 5000
-          }))
-          .pipe(istanbul.writeReports())
-          .on('finish', function() {
-            return del(paths.testBuild).then(exit);
-          });
-      });
-  })
-});
-
 // Re-runs the linter and regenerates js every time a source file changes
```

Comment:
This doesn't seem to run the linter anymore.

Reply:
Updated comment.